snow/consensus/snowstorm/directed.go

```diff
@@ -216,8 +219,8 @@ func (dg *Directed) RecordPoll(votes ids.Bag) (bool, error) {
 
 		txNode.RecordSuccessfulPoll(dg.currentVote)
 
-		dg.ctx.Log.Verbo("Updated TxID=%s to have consensus state=%s",
-			txID, &txNode.snowball)
+		dg.ctx.Log.Verbo("Updated TxID=%v to have consensus state=%s",
+			txIDKey, &txNode.snowball)
```

Comment:
Isn't this going to cause a ton of memory allocs like the other log?

Reply:
yeah I meant to remove this